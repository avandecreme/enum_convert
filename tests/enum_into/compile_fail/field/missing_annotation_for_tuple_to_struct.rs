use enum_convert::EnumInto;

#[derive(EnumInto)]
#[enum_into(Target)]
enum Source {
    #[enum_into(Target::Struct)]
    Tuple(
        // #[enum_into(Target::Struct.b)] missing
        i32,
        #[enum_into(Target::Struct.a)] i32,
    ),
}

enum Target {
    Struct { a: i64, b: i32 },
}

fn main() {}
