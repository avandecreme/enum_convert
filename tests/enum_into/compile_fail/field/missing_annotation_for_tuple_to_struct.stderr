error: Missing required mapping to named field for Target::Struct
 --> tests/enum_into/compile_fail/field/missing_annotation_for_tuple_to_struct.rs:9:9
  |
9 |         i32,
  |         ^^^
